designsafe/static/scripts/data-depot/components/projects/curation-directory/curation-directory.component.html

```diff
@@ -4,8 +4,8 @@
         <div class="prj-head-container">
             <span class="prj-head-title">
                 {{$ctrl.browser.project.value.projectId}} | <strong>{{ $ctrl.browser.project.value.title }}</strong>
-                <a class="prj-head-edit" ng-click="$ctrl.manageProject($event)"><strong>Edit Project</strong></a>
             </span>
+            <a class="prj-head-edit" ng-click="$ctrl.manageProject($event)"><strong>Edit Project</strong></a>
```

Comment:
In the components for each publication preview we can remove the `manageProject()` functions that call those modals since we're not using them anymore.